packages/larva-css/src/02-algorithms/a-hover-effect/a-hover-effect.common.async.scss

```diff
@@ -23,3 +23,7 @@
 
 	border-bottom: 1px solid currentColor;
 }
+
+.lrv-a-hover-effect-text-decoration--underline:hover {
```

Comment:
We don't use `--` in the class names in our naming conventions. Just `lrv-a-hover-effect-text-decoration-underline` is good.

Reply:
Also - just realized this already exists as `lrv-u-text-decoration-underline:hover`